manifests/application.pp

```diff
@@ -3,12 +3,8 @@
 # @param rest_client_timeout
 #   Timeout for Katello rest API
 #
-# @param repo_export_dir
-#   Create a repository export directory for Katello to use
-#
 class katello::application (
   Integer[0] $rest_client_timeout = 3600,
```

Comment:
Unrelated, but I think this is also a good candidate to move to a regular setting that can be changed at runtime. Do you agree and should I create an issue for it?

Reply:
Let's see what @jlsherrill  thinks